modules/user_accounts/ajax/rejectUser.php

```diff
@@ -1,122 +1,81 @@
 <?php
 /**
- * Media uploader.
+ * User accounts
  *
- * Handles media upload and update actions received from a front-end ajax call
+ * Handles rejection of pending user accounts by a user that has permissions
  *
  * PHP Version 5
  *
  * @category Loris
  * @package  User_Accounts
  * @author   Leo T. <lthomas.mcin@gmail.com>
+ *           Zaliqa Rosli <zaliqa.rosli@mcin.ca>
  * @license  Loris license
  * @link     https://github.com/aces/Loris
  */
 
-    define('NO_IDENTIFIER_SUPPLIED', 1);
-    define('USER_NOT_FOUND', 2);
-    define('INCORRECT_PERMISSION', 3);
-    define('ACCOUNT_ACTIVE', 4);
+use LORIS\user_accounts as UA;
 
+define('NO_IDENTIFIER_SUPPLIED', 1);
+define('USER_NOT_FOUND', 2);
+define('INCORRECT_PERMISSION', 3);
+define('ACCOUNT_ACTIVE', 4);
 
-if (!isset($_POST['identifier'])) {
+if (isset($_POST['identifier'])) {
+    $identifier = $_POST['identifier'];
+    _rejectUser($identifier);
+} else {
     throw new LorisException("No identifier supplied", NO_IDENTIFIER_SUPPLIED);
     exit(1);
```

Comment:
this is unreachable.

Reply:
@driusan could you elaborate please?

Reply:
it's impossible to reach the line "exit(1)", because the line immediately above it unconditionally throws an exception. The exit should just be removed.